src/basis/template/namespace.js

```diff
@@ -0,0 +1,31 @@
+var namespaceURI = {
+  xlink: 'http://www.w3.org/1999/xlink',
+  svg: 'http://www.w3.org/2000/svg'
+};
+
+function getTagNamespace(name){
+  var colonIndex = name && name.indexOf(':');
+  if (colonIndex != -1)
+  {
+    var prefix = name.substr(0, colonIndex);
+    return namespaceURI[prefix];
+  }
+}
+
+function getAttributeNamespace(name, node){
```

Comment:
I suppose that two similar function is not needed. `resolveNamespace(name, node)` is enough, where `node` is optional argument.